app/src/routes/ChoiceBoard.tsx

```diff
@@ -14,9 +13,16 @@ const H3 = styled.h3`
   color: #2e2a27;
 `;
 
+
 function ChoiceBoard() {
   const { t } = useTranslation();
 
+  const commonOnClick = (e, card) => {
+    console.log("Card clicked!!")
+    console.log(card)
+  }
+
+
   return (
     <Suspense fallback="loading">
```

Comment:
Do we need two `Suspense`, one here and another one wrapping `ChoiceBoard` in `app/src/App.tsx`?

Reply:
I think we don't need `<Suspense />` right now. We can discuss this in a future issue.